herd/event.ml

```diff
@@ -198,14 +198,21 @@ val same_instance : event -> event -> bool
   and module Elts = EventSet
 
   type event_rel = EventRel.t
+  type event_trans_rel = { input: EventSet.t; rel: EventRel.t; output: EventSet.t }
```

Comment:
As there are other **input** and **output** record fields in this file, compilation with dune **release** profile yields a lot of warnings. I am in favor of renaming those fields as  **trans_input** and **trans_output** for instance.
Alternatively:
```suggestion
module Trans = struct
  type t = { input: EventSet.t; rel: EventRel.t; output: EventSet.t }
end
```
Then the fields are `Trans.input`, `Trans.output`, etc.